.github/workflows/rebuild-released-images.yaml

```diff
@@ -90,7 +98,7 @@ jobs:
           context: .
           build-args: VERSION=${{ matrix.version }}
           platforms: ${{ fromJSON(needs.read-versions.outputs.matrix).platforms }}
-          tags: ${{ env.IMAGE_REPOSITORY }}:${{ steps.daily-tag.outputs.daily-tag }}
+          tags: ${{ github.event.inputs.image_repo }}:${{ steps.daily-tag.outputs.daily-tag }}
```

Comment:
What's gonna be this value when triggered by the scheduled job?

Reply:
Default value **mongodb/mongodb-atlas-kubernetes-operator**
```yaml
  inputs:
      image_repo:
        type: choice
        description: "Target image repository for built images"
        default: mongodb/mongodb-atlas-kubernetes-operator
```

Reply:
we agreed on making the prereleases repo the default, but then I also need to tweak the schedule to run against the production repo.

Reply:
done